src/services/ContentService.js

```diff
@@ -53,14 +53,16 @@ define(["structures/ContentCreateStruct", "structures/ContentUpdateStruct", "str
      * @method loadRoot
      * @param callback {Function} callback executed after performing the request (see
      * {{#crossLink "ContentService"}}Note on the callbacks usage{{/crossLink}} for more info)
+     * @param xhrCallbacks {Object} a set of callbacks to apply on a specific XHR event like onload, onerror, onprogress, etc.
      */
-    ContentService.prototype.loadRoot = function (callback) {
+    ContentService.prototype.loadRoot = function (callback, xhrCallbacks) {
```

Comment:
ah you did change every method ?
this is not really necessary, it makes very little sense for most of them (this one is a good example of that). Also that means you have to add a unit test for each method using the new parameter to validate it's actually pass the ConnectionManager. So as written in https://github.com/ezsystems/ez-js-rest-client/pull/82#issuecomment-285629578 a first step, I would do that only for the method where you need that feature or on methods where it makes sense. If it's missing on a method, it can be easily added later.

But then same remark as above on the name of the parameter, its documentation as optional, the order. Also you should add an example as well but not on each method where you add that feature.